projects/authentication/docs/environments.md

```diff
@@ -0,0 +1,30 @@
+# Authorization server URLs
+
+The authorization server to request [tokens](./methods.md) on is available on two domains, one for the test environment and one for the production environment of our APIs.
```

Comment:
```suggestion
The authorization server to request [tokens](./methods.md) is available on two domains, one for the test environment and one for the production environment of our APIs.
```

Reply:
Hmm I meant it as in "requesting tokens on the server" -> "the server to request tokens on". But if it reads weird or is incorrect maybe it's better to reword the whole sentence 🤔

Reply:
@corneelwille can you check the change in [c8e359f](https://github.com/cultuurnet/apidocs/pull/64/commits/c8e359f519bc089da1082dda54c7eb53a550b8be)?